pkg/config/config_test.go

```
@@ -0,0 +1,66 @@
+	if err != nil {
+		t.Fatalf("Failed to create temp directory: %v", err)
+	}
+	defer os.RemoveAll(tmpDir)
+	// create atmos.yaml file
+	atmosConfigFilePath = filepath.Join(tmpDir, atmosConfigFilePath)
+	f, err := os.Create(atmosConfigFilePath)
+	if err != nil {
+		t.Fatalf("Failed to create atmos.yaml file: %v", err)
+	}
+	// write atmos.yaml content
+	f.WriteString("base_path: ./ \n")
```

golangci-lint (1.62.2)

Error return value of `f.WriteString` is not checked

(errcheck)